src/pages/ModalPage.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import DefaultInternalTemplate from "../templates/DefaultInternalTemplate";
+import {
+  Breadcrumbs,
+  BreadcrumbItem,
+  BreadcrumbLinkItem,
+  IconHeading,
+  IconButton,
+  Section,
+  SummaryList
+} from "@baltimorecounty/dotgov-components";
+
+const sideBarContent = [
```

Comment:
would be good to move this into it's own component, since this is repeated so much.

Reply:
this was addressed, other pages will be updated as they are updated